rke/k8s_rke_system_images.go

```diff
@@ -7142,6 +7142,92 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.18.0",
 		},
+		"v1.21.12-rancher1-2": {
```

Comment:
Since `v1.21.13-rancher1-1` based on k8s 1.21.13 is already released, we would generally not introduce `v1.21.12-rancher1-2` based on not the latest patch. Is there a reason this updated ACI can't be based on 1.21.13?

In fact, for the upcoming release we're introducing k8s June patch versions - https://github.com/rancher/kontainer-driver-metadata/pull/921. So realistically we would want to introduce updated ACI images in the versions to be added per that PR: `1.23.8-rancher1-1`, `1.22.11-rancher1-1`, `1.21.14-rancher1-1`. The best option would be to wait for the linked PR to be merged and then rebase your PR and make updates to `1.23.8-rancher1-1`, `1.22.11-rancher1-1`, `1.21.14-rancher1-1`. In this case making updates to already existing versions is an acceptable process as these versions were not released yet (not merged to `release-v2.6` branch).